### PR TITLE
feat: Add functions_firebase_rtdb sample

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

**Review by ace-n — APPROVED** [review]
LGTM if this works on Cloud Functions itself. 🙂

File: functions/test/firebase/rtdb_test.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      payload = {
+        "params" => { "child" => "abcde" },
+        "delta"  => { "foo" => "bar" }
+      }
+      event = make_cloud_event payload, source: source, type: type
+      _out, err = capture_subprocess_io do
+        call_event "hello_rtdb", event
+      end
+
+      assert_includes err, "Function triggered by change to: #{source}"
```
Comment (ace-n) [inline functions/test/firebase/rtdb_test.rb:34]: Nit: should this be `out`? (Unless printing to `stderr` is more idiomatic in Ruby.)

File: functions/firebase/rtdb/app.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_firebase_rtdb]
+require "functions_framework"
+
+# Triggered by a change to a Firebase RTDB document.
+FunctionsFramework.cloud_event "hello_rtdb" do |event|
+  # The event parameter is a CloudEvents::Event::V1 object.
```
Comment (ace-n) [inline functions/firebase/rtdb/app.rb:20]: I'm concerned that users may interpret this as being a format _specific_ to Firebase, as opposed to something shared across multiple triggers.

Maybe re-word to `Non-HTTP Ruby functions receive a CloudEvents::Event::V1 object.`?

**Review by ace-n — CHANGES_REQUESTED** [review]

File: functions/test/firebase/rtdb_test.rb (hunk 1 above)
Comment (dazuma) [reply functions/test/firebase/rtdb_test.rb:34]: The Ruby functions framework logs to stderr by default. This is standard behavior for Ruby web servers.

Comment (ace-n) [reply functions/test/firebase/rtdb_test.rb:34]: Ah OK, noted.

File: functions/firebase/rtdb/app.rb (hunk 2 above)
Comment (dazuma) [reply functions/firebase/rtdb/app.rb]: Updated

**Pull request merged**